### PR TITLE
save new payload into 'payload' field

### PR DESCRIPTION
**Pull request by cob16**
The new payload needs to be saved as currently only the submittion details are saved.

### CODE REVIEW

File: app/controllers/submission_controller.rb [hunk 1]
```diff
@@ -20,14 +21,22 @@ def show
...
-  def submission_params(opts = params)
+  def submission_params
     # we must use slice(..).permit! rather than permitting individual params, as
     # submission_details is an arbitrary hash, which AC Strong Params *really*
     # doesn't like
-    opts.slice(
+    params.slice(
       :service_slug,
       :encrypted_user_id_and_token,
       :submission_details
```
Comment (cob16) [inline app/controllers/submission_controller.rb]: this will go once no longer needed

File: app/controllers/submission_controller.rb [hunk 2]
```diff
@@ -1,10 +1,11 @@
 class SubmissionController < ApplicationController
   def create
-    @submission = Submission.new(
-      submission_params(params).merge(status: Submission::STATUS[:queued])
+    @submission = Submission.create(
+      submission_params.merge(
+        status: Submission::STATUS[:queued],
```
Comment (asmega) [inline app/controllers/submission_controller.rb]: not one for this PR but it maybe that `status` is not being used

File: app/controllers/submission_controller.rb [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
+        payload: payload
+      )
     )
-    @submission.save!
```
Comment (asmega) [inline app/controllers/submission_controller.rb]: the bang operator disappeared. so `id` of submission could be `nil`. is that desired?

File: app/controllers/submission_controller.rb (hunk 2 above)
Comment (cob16) [reply app/controllers/submission_controller.rb]: agreed, good to kill at the same time as submitton_details

File: app/controllers/submission_controller.rb (hunk 3 above)
Comment (cob16) [reply app/controllers/submission_controller.rb]: i used the create method so it will explode on the line before insted

Comment (cob16) [reply app/controllers/submission_controller.rb]: fixed

**Pull request merged**